Apps/System/notification.c

```diff
@@ -18,7 +18,7 @@ static NotificationWindow *notif_window;
 
 void notif_init(void)
 {
-    printf("init\n");
+    SYS_LOG("notification", APP_LOG_LEVEL_INFO, "init");
```

Comment:
We probably should use `APP_LOG` in the system apps, but idk if the macro is already copied from the original `pebble.h` file.

Reply:
Agreed. Can you PR it or add an issue.

Reply:
Created issue pebble-dev/RebbleOS/issues/39